code/hud/hudbrackets.cpp

```diff
@@ -395,7 +395,7 @@ void HudGaugeBrackets::renderObjectBrackets(object *targetp, color *clr, int w_c
 	int bound_rc;
 	SCP_list<CJumpNode>::iterator jnp;
 
-	if ( Player->target_is_dying <= 0 ) {
+	if ( Player->target_is_dying <= 0 || targetp != &Objects[Player_ai->target_objnum]) {
```

Comment:
Is there some guarantee that Player_ai->target_objnum won't be -1 or otherwise invalid?